### ISSUE TITLE
Test with NA 3km grid on wcoss2

### ISSUE THREAD

**Issue by hu5970** [NOAA-EMC/rrfs-workflow#55]
This document issues we found in the test.

**Comment by hu5970** [comment]
Set "`PREEXISTING_DIR_METHOD="rename"`"

**Comment by hu5970** [comment]
All RRFS_A configure files including EnKF DA and ensemble forecast should be in together.

**Comment by hu5970** [comment]
need to edit "run_recenter.local.lua"

**Comment by chan-hoo** [comment]
@hu5970, can you explain again what is the difference between `RRFS_A` and `RRFS_B`?

**Comment by chan-hoo** [comment]
@hu5970, which sample script can I use to test `run_recenter.local`, `enkf_radarref`, and `run_enkfupdt.local`?

**Comment by hu5970** [comment]
@chan-hoo I have a test system running on Dogwood. It is v0.7.0 in the same location with other RRFS A runs. I cannot login WCOSS2 in next two weeks. Could you find my test system and find the "run_recenter.local, enkf_radarref, and run_enkfupdt.local". Ben should be able to help to find the location of my test on Dogwood. Otherwise, your can use GSI run environment for "run_recenter.local, enkf_radarref, and run_enkfupdt.local".

**Comment by hu5970** [comment]
@chan-hoo on RRFS_A and B. They are the same but configured differently because they are running on different machines with different resources. RRFS_A runs on WCOSS2 with full set of configurations for future operation. RRFS_B runs on JET with very limited resources. Right now, RRFS_B runs over CONUS 3km with the deterministic cycle only. RRFS_B is mainly used for testing new functions before they go into RRFS_A. So, please focus on RRFS_A.

**Comment by hu5970** [comment]
@BenjaminBlake-NOAA Ben, I cannot access WCOSS2 in next two weeks. But I set up a testing RRFS_A system for rrfs-workflow yesterday. It is called v0.7.0 in the same location with other RRFS_A system on Dogwood. Could you check it when you have time?It should run the same way as v0.6.7 on Cactus.  If it runs OK in next one or two days, you can turn it off. Thanks, Ming

**Comment by chan-hoo** [comment]
@hu5970, in "sample_configs", I created `ensemble` and `others` as below:
```
- ensemble: RRFS_CONUS_13km_ens, RRFS_CONUS_3km_ens
- others: con: RRFS_AK_dev1, RRFS_CONUS_13km, RRFS_CONUS_3km, RRFS_NA_13km, RRFS_NA_3km
```
Are they good as it is? or should I re-organize them too?

**Comment by chan-hoo** [comment]
@BenjaminBlake-NOAA, can you point me to the working directory on WCOSS2 (Dogwood) that @hu5970 mentioned above?

**Comment by BenjaminBlake-NOAA** [comment]
@hu5970  Sure I will keep an eye on it and turn it off in the next day or two assuming it looks good.  It is just the deterministic forecast + EnKF running right?  Looks like the ensemble forecasts are not running.

**Comment by hu5970** [comment]
@chan-hoo From my point of view, the deterministic cycle and ensemble cycles are all belongs to RRFS system. So, I think we can put all RRFS_A configures under one sub-directory. from my case (v0.7.0), you can see my put both deterministic and ensemble DA to RRFS_A. (ensemble forecast uses the same configure scripts with ensemble DA).

**Comment by hu5970** [comment]
@BenjaminBlake-NOAA I did configured ensemble forecast and hope it can run since yesterday 00Z. It is OK if it did not run. We should be able to fix ensemble forecast fairly easily. Main tests should be DA cycles.

**Comment by BenjaminBlake-NOAA** [comment]
@hu5970  I agree, the main tests should be for the deterministic forecast + EnKF.  It looks like the ensemble forecast is commented out in emc.lam's crontab under the v0.7.0 section

**Comment by chan-hoo** [comment]
> @chan-hoo From my point of view, the deterministic cycle and ensemble cycles are all belongs to RRFS system. So, I think we can put all RRFS_A configures under one sub-directory. from my case (v0.7.0), you can see my put both deterministic and ensemble DA to RRFS_A. (ensemble forecast uses the same configure scripts with ensemble DA).

@hu5970, I'll put the scripts in `EnKF` into `RRFS_A`. However, the scripts I mentioned above are not for `wcoss2`. They are for `jet` or `hera`. I'll move two scripts in `ensemble` into `others`. Eventually, we'll have `RRFS_A`, `RRFS_B`, `RTMA`, and `others` (besides eng tests). Is this ok?

**Comment by hu5970** [comment]
@chan-hoo That sounds good.

**Comment by hu5970** [comment]
@BenjaminBlake-NOAA  I may forgot to turn on crontab for ensemble forecast in v0.7.0. If so, please turn it on and we just need to test one cycle. Any one works.

**Comment by BenjaminBlake-NOAA** [comment]
@hu5970  Sure I turned on the crontab entry and will monitor it for one cycle.  The deterministic forecast and EnKF seem to be working well so I will turn those off later today for v0.7.0.

**Comment by BenjaminBlake-NOAA** [comment]
@hu5970 @chan-hoo  The product generation job is not working for the v0.7.0 test.  It cannot find the mpiexec command.  I think the craypals module is not being loaded.  Should this be added to tasks/wcoss2/run_prdgen.local.lua or somewhere else?  Otherwise v0.7.0 seems to be running well.

**Comment by chan-hoo** [comment]
@BenjaminBlake-NOAA, can you add the following modules to `tasks/wcoss2/run_prdgen.local.lua` and re-run the task again?
```
load(pathJoin("intel", os.getenv("intel_ver")))
load(pathJoin("PrgEnv-intel", os.getenv("PrgEnv_intel_ver")))
load(pathJoin("craype", os.getenv("craype_ver")))
load(pathJoin("cray-mpich", os.getenv("cray_mpich_ver")))
load(pathJoin("cray-pals", os.getenv("cray_pals_ver")))
load(pathJoin("hdf5", os.getenv("hdf5_ver")))
load(pathJoin("netcdf", os.getenv("netcdf_ver")))

load(pathJoin("udunits", os.getenv("udunits_ver")))
load(pathJoin("gsl", os.getenv("gsl_ver")))
load(pathJoin("nco", os.getenv("nco_ver")))
```

**Comment by BenjaminBlake-NOAA** [comment]
@chan-hoo  Thanks!  I just updated run_prdgen.local.lua.  Various run_prdgen tasks are in the process of being submitted so I will monitor them.

**Comment by BenjaminBlake-NOAA** [comment]
@chan-hoo That fixed the issue with the run_prdgen task.  Can we get that added to PR #56 ?

**Comment by BenjaminBlake-NOAA** [comment]
I went ahead and turned off the v0.7.0 test on Dogwood.  I didn't see any other issues.

**Comment by chan-hoo** [comment]
@BenjaminBlake-NOAA, added!

**Comment by MatthewPyle-NOAA** [comment]
This issue is a bit old, and since we're definitely running with the NA 3 km grid on WCOSS2, guessing it could be closed?

**Comment by chan-hoo** [comment]
Yes, I think so, too.

**Comment by BenjaminBlake-NOAA** [comment]
Agreed about closing this issue.
